match.go

```diff
@@ -3,198 +3,39 @@ package sftp
 import (
 	"path"
 	"strings"
-	"unicode/utf8"
 )
 
 // ErrBadPattern indicates a globbing pattern was malformed.
 var ErrBadPattern = path.ErrBadPattern
 
-// Unix separator
-const separator = "/"
-
-// Match reports whether name matches the shell file name pattern.
-// The pattern syntax is:
-//
-//	pattern:
-//		{ term }
-//	term:
-//		'*'         matches any sequence of non-Separator characters
-//		'?'         matches any single non-Separator character
-//		'[' [ '^' ] { character-range } ']'
-//		            character class (must be non-empty)
-//		c           matches character c (c != '*', '?', '\\', '[')
-//		'\\' c      matches character c
-//
-//	character-range:
-//		c           matches character c (c != '\\', '-', ']')
-//		'\\' c      matches character c
-//		lo '-' hi   matches character c for lo <= c <= hi
-//
-// Match requires pattern to match all of name, not just a substring.
-// The only possible returned error is ErrBadPattern, when pattern
-// is malformed.
-//
+// Match reports whether name matches the shell pattern.
 //
+// This is an alias for path.Match from the standard library, offered so
+// that callers need not import the path package. For details, see
+// https://golang.org/pkg/path/#Match.
```

Comment:
[nitpick] Not in any way an actual issue, but I tend to recommend https://rhodesmill.org/brandon/2012/one-sentence-per-line/

I mention this really only because in this particular case splitting this on semantic boundaries gets nearly the same line breaks, but split up the text more coherently:
```
// This is an alias for path.Match from the standard library,
// offered so that callers need not import the path package.
// For details, see https://golang.org/pkg/path/#Match.
```

If you don’t want to fix this nitpick, just say so, and I will merge anyways.

Reply:
Done.